recipes/qt/6.x.x/conanfile.py

```diff
@@ -211,11 +211,11 @@ def _enablemodule(mod):
     def requirements(self):
         self.requires("zlib/1.2.11")
         if self.options.openssl:
-            self.requires("openssl/1.1.1j")
+            self.requires("openssl/1.1.1k")
         if self.options.with_pcre2:
             self.requires("pcre2/10.36")
         if self.options.with_vulkan:
-            self.requires("vulkan-loader/1.2.172.0")
+            self.requires("vulkan-loader/1.2.170.0")
```

Comment:
`vulkan-loader/1.2.172` exists (without `.0`). Maybe use that instead?

Reply:
I don't mind, but [@SpaceIm prefers sdk versions](https://github.com/conan-io/conan-center-index/issues/5371#issuecomment-829926150)